requirements.txt

```diff
@@ -38,7 +38,7 @@ ujson==1.35
 urllib3==1.22
 venusian==1.1.0
 waitress==1.1.0
-WebOb==1.7.4
+WebOb==1.8.0rc1
```

Comment:
Really?

Reply:
Hmm, oops? I just did `make build-requirements` and committed without really reviewing it. I'll roll it back..

Reply:
No worries just checking :)